api/GPU.json

```diff
@@ -14,9 +14,13 @@
           },
           "edge": "mirror",
           "firefox": {
+            "version_added": "preview",
+            "partial_implementation": true,
+            "notes": "Currently supported on Linux and Windows only. See <a href='https://bugzil.la/1616739'>bug 1616739</a>."
```

Comment:
If I care about macOS support, I can't really see anything in the linked bug. Is there a bug that tracks that specifically? Otherwise I think we should remove the link.

Reply:
Can't find one. Stripping out the link.